src/api/requestsApi.ts

```diff
@@ -62,9 +83,17 @@ export const useRequests = () => {
 };
 
 const getRequests = async () => {
+  // Grab all fields, and expand Person fields
   const requestedFields =
-    "*,orgApprover/Id,orgApprover/EMail,orgApprover/Title";
-  const expandedFields = "orgApprover";
+    "*," +
+    "orgApprover/Id,orgApprover/EMail,orgApprover/Title," +
+    "supervisor/Id,supervisor/EMail,supervisor/Title," +
+    "organizationalPOC/Id,organizationalPOC/EMail,organizationalPOC/Title," +
+    "issueTo/Id,issueTo/EMail,issueTo/Title";
+
+  const expandedFields = "orgApprover,supervisor,organizationalPOC,issueTo";
+
+  //
   return spWebContext.web.lists
```

Comment:
We don't call this yet, but guessing we are going to want to restrict this query to only return then RPADocSet content type, and not all items